backend/pom.xml

```diff
@@ -77,7 +77,7 @@
 		<dependency>
       <groupId>org.springdoc</groupId>
       <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
-      <version>2.0.0</version>
+      <version>2.0.2</version>
```

Comment:
Same update is being handled on PR #193 

Please do not merge this, instead close this PR and follow up with the above one.